apps/web/src/views/Farms/v4/components/PoolListView.tsx

```diff
@@ -0,0 +1,202 @@
+              quoteToken={item.token1}
+              width="48px"
+              icon={
+                <TokenPairImage
+                  width={44}
+                  height={44}
+                  variant="inverted"
+                  primaryToken={item.token0}
+                  secondaryToken={item.token1}
+                />
+              }
+            />
+            <div>{(Number(item.lpApr) * 100).toLocaleString(undefined, { maximumFractionDigits: 2 })}%</div>
+          </Column>
+
+          <Column>
+            <Button scale="xs" variant="text" onClick={() => setOpenItem(item)}>
+              <MoreIcon />
+            </Button>
+          </Column>
+        </ListItemContainer>
+      ))}
+
+      <BottomDrawer
+        drawerContainerStyle={{ height: 'auto' }}
+        isOpen={openItem !== null}
+        setIsOpen={handleDrawerChange}
+        content={drawerContent}
+      />
+    </ListContainer>
+  )
+}
+
+const ItemDetailContainer = styled.div``
+
+const ItemDetailHeader = styled.div`
+  display: flex;
+  padding: 16px 0 8px;
+  justify-content: center;
+`
+
+const Grabber = styled.div`
+  width: 36px;
+  height: 4px;
+  border-radius: 9999px;
+  opacity: 0.1;
+  background: ${({ theme }) => theme.colors.contrast};
+`
+const ItemDetailFooter = styled.div`
+  border-top: 1px solid ${({ theme }) => theme.colors.cardBorder};
+`
+
+export const StyledButton = styled(Button)`
+  display: flex;
+  padding: 12px 16px;
+  justify-content: space-between;
+  gap: 8px;
+  align-self: stretch;
+  color: ${({ theme }) => theme.colors.text};
+  font-weight: 400;
+  line-height: 24px;
+`
+
+const ItemDetailBody = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: flex-start;
+  gap: 8px;
+  padding: 16px;
+  color: ${({ theme }) => theme.colors.textSubtle};
+  font-size: 12px;
+  font-weight: 600;
+`
+
+const ListItem = styled.div`
+  display: flex;
+  width: 100%;
+  justify-content: space-between;
+  align-items: flex-end;
+  line-height: 150%;
+`
+
+const ListItemLabel = styled(Column)`
+  text-transform: uppercase;
+`
+
+export interface IListItemDetailsProps {
+  data: PoolInfo | null
+}
+
+const ListItemDetails: React.FC<IListItemDetailsProps> = ({ data }) => {
+  const { t } = useTranslation()
+  const columns = useColumnMobileConfig<PoolInfo>()
+
+  if (!data) {
+    return null
+  }
+
+  return (
+    <ItemDetailContainer>
+      <ItemDetailHeader>
+        <Grabber />
+      </ItemDetailHeader>
+      <ItemDetailBody>
+        <TokenOverview
+          isReady
+          token={data.token0}
+          quoteToken={data.token1}
+          width="48px"
+          icon={
+            <TokenPairImage
+              width={44}
+              height={44}
+              variant="inverted"
+              primaryToken={data.token0}
+              secondaryToken={data.token1}
+            />
+          }
+        />
+        {columns
+          .filter((col) => col.dataIndex !== null)
```

Comment:
useMemo for filter result

Reply:
i have separate a new columns config for mobile, so this filter can just drop. fixed